packages/api/src/brevo-contact/brevo-contact.console.ts

```diff
@@ -0,0 +1,53 @@
+import { CreateRequestContext, MikroORM } from "@mikro-orm/core";
+import { Injectable } from "@nestjs/common";
+import { Command, Console } from "nestjs-console";
+
+import { BrevoApiContactsService } from "../brevo-api/brevo-api-contact.service";
+import { TargetGroupsService } from "../target-group/target-groups.service";
+
+@Injectable()
+@Console()
+export class DeleteUnsubscribedContactsConsole {
+    constructor(
+        private readonly brevoApiContactsService: BrevoApiContactsService,
+        private readonly targetGroupsService: TargetGroupsService,
+        private readonly orm: MikroORM,
+    ) {}
+
+    @Command({
+        command: "delete-unsubscribed-contacts",
+        description: "deletes unsubscribed contacts",
+    })
+    @CreateRequestContext()
+    async execute(): Promise<void> {
+        const offset = 0;
+        const limit = 50;
+        const where = { isMainList: true };
+
+        const [targetGroups] = await this.targetGroupsService.findMainTargetGroups({ offset, limit, where });
+
+        for (const targetGroup of targetGroups) {
+            let numberOfBlacklistedContacts = false;
```

Comment:
As the variable naming suggests I would assume this should be a number. 
So this should always be a number and be `0` here